dependencies.md

```diff
@@ -0,0 +1,73 @@
+Dependencies
+============
+
+The software we develop often depends on standalone packaged libraries.
+Managing these dependencies is crucial to delivery: we need processes that yield
+consistent results in all environments, whether on a developer’s machine or a
+deploys. Most languages have package manager which aim to solve this issue.
+
+## [Semantic Versioning][semver]
+
+> Given a version number `major`.`minor`.`patch`, increment the:
+>
+> - `major` version when you make incompatible API changes;
+> - `minor` version when you add functionality in a backwards compatible manner;
+> - `patch` version when you make backwards compatible bug fixes.
+>
+> Additional labels for pre-release and build metadata are available as extensions to the `major`.`minor`.`patch` format.
+
+Several tools, such as [Dependabot][] or Snyk, can automatically and regularly
```

Comment:
Totally a style thing, but I don't think you need the empty `[]` here.

Ditto the `[Semantic Versioning][semver]` link could become just `[Semantic Versioning]` and then link with `[Semantic Versioning]: https://semver.org/`.

Reply:
It works “magically” in Github, but omitting the `[]` [isn’t in the Markdown spec] [1]. I slighlty prefer the more verbose and explicit and don’t find it impedes readability.

[1]: https://daringfireball.net/projects/markdown/syntax#link

Reply:
Ahh TIL, thanks for explaining 👍